### ISSUE TITLE
It's not 100% clear this is the right place to get RimPy for first time users.

### ISSUE THREAD

**Issue by origintopleft** [rimpy-custom/RimPy#127]
My girlfriend's been getting me into Rimworld, and she's been trying to get me to install RimPy to use some of the performance related tools (something about converting texture files to DDS and an automatic sorter that sounds kinda like LOOT from Bethesda modding?). My first instinct is to Google for the tool, and I got this repository as the first result...but it's just a template-generated README and a PNG file. I thought I got a bogus Google result. Luckily, the Steam Workshop mod was also on the first page and linked back here, but to the releases page where the downloads are actually available.

I'm assigning the actual blame on this to GitHub not being 100% meant for proprietary software to host downloads, and therefore its UX isn't 100% up to the task (I don't personally think to check if there's releases if I don't see code in the repo). A common practice, however, is to extend the generated README.md to point to the right places, just so folks landing here from search engines know they haven't misclicked.

(I would have made a pull request and fixed the README myself, but you don't seem to keep your assets for your Steam workshop description in this repo and I would have been too perfectionist about trying to make a "Workshop Link" header that matched)

**Comment by Lucas559-noob** [comment]
Well, I think GitHub is the best place for direct downloads(without having the chance to download an ad) and help(Issues) or improvements(fork and PR)

**Comment by King-of-Blades** [comment]
I mean, your failure to notice the releases section highlighted in bright colors just slightly off to the right of center isn't really his issue, not to be rude.  For someone who seems familiar with github, I fail to understand why you wouldn't notice that there are releases present.

**Comment by origintopleft** [comment]
> I mean, your failure to notice the releases section highlighted in bright colors just slightly off to the right of center isn't really his issue, not to be rude. For someone who seems familiar with github, I fail to understand why you wouldn't notice that there are releases present.

It's not "duh bhuh wher r download" so much, it was merely the empty `README.md` file combined with "rimpy-*custom*" which made me unsure whether I had the upstream RimPy repository in front of me or a third party fork. 

Also, I'd like to apologize if I'm coming off rude or impolite or anything, Feb 20th was roughly when I caught the rona and it seems to have caused some kind of permanent emotional instability and trouble forming logical sentences that I'm still struggling with even now. Hopefully, I'm managing it a little better now, but getting it looked at by a neurologist or something is on my to-do list.

If this is still considered a waste of everyone's time, I have no problem with closing this issue.

**Comment by char-cole** [comment]
not everyone is familiar with github though - a simple readme would be a valuable accessibility feature, if that helps. it really isn't super clear that this is the correct repo.

**Comment by almera-vs** [comment]
> not everyone is familiar with github though - a simple readme would be a valuable accessibility feature, if that helps. it really isn't super clear that this is the correct repo.

You're on github. You can contribute whenever you want

**Comment by LionelColaso** [comment]
yeah its funny this is a closed source project and how do u expect people to contribute

**Comment by KurtSlyCamilleri** [comment]
Yeah, just got lead here also, no clue what to do next and how to download now, I'm just not gonna use it *shrug*

**Comment by almera-vs** [comment]
Here's a quick guide since people new to github have problems:

**If you're on Windows:**

1. Download last windows release [here](https://github.com/rimpy-custom/RimPy/releases/download/1.2.6.28/RimPy_Windows.zip)
2. unzip it
3. launch the exe file

**If you're on Linux:**

1.  Download last linux release [here](https://github.com/rimpy-custom/RimPy/releases/download/1.2.6.29/RimPy_Linux.tar.gz)
2.  unpack the file (using archive app or by command `tar -xvf RimPy_Linux.tar.gz`)
3.  go to terminal and type `sudo ./RimPy.sh` while inside the unpacked directory

**Comment by Tigeress4** [comment]
> Here's a quick guide since people new to github have problems:
> 
> **If you're on Windows:**
> 
> 1. Download last windows release [here](https://github.com/rimpy-custom/RimPy/releases/download/1.2.6.28/RimPy_Windows.zip)
> 2. unzip it
> 3. launch the exe file
> 
> **If you're on Linux:**
> 
> 1. Download last linux release [here](https://github.com/rimpy-custom/RimPy/releases/download/1.2.6.29/RimPy_Linux.tar.gz)
> 2. unpack the file (using archive app or by command `tar -xvf RimPy_Linux.tar.gz`)
> 3. go to terminal and type `sudo ./RimPy.sh` while inside the unpacked directory

Hello and thank you for this, I just made an account, 1st time here. So I apologize if I'm somehow in the wrong asking, but I found this from steam workshop and recommendations on Reddit, but I play I steam deck. I don't understand how to get  this loaded on my deck and if maybe its not needed for the deck? My kid was into Rimworld but saved borked so I started doing research and playing a bit to help (got into it myself I'm normally a Sims' player)  realized most people said it was Prepare carefully was most likely to blame so now trying to go over this 12 steam pages of subscribed mods to "fix the list" and I admittedly jumped out of the frying pan and into the pasta water... 

I am used to moding Sims games, but I am NOT a coder please explain things to me in a cross between Senior citizen, a 6-8 (1st or 2nd grade) and kitten or puppy (which ever your fav is I promise I have permission to be on the chair/ keyboard :)! )

**Comment by Lucas559-noob** [comment]
Steam deck is like Linux or how? Maybe isn't supported

Em ter, 1 de ago de 2023 13:25, Tigeress4 ***@***.***>
escreveu:

**Comment by King-of-Blades** [comment]
> Steam deck is like Linux or how? Maybe isn't supported


Steam deck is Linux. It's most definitely supported, but you have to set it up through the desktop mode and it's not easy to do for someone to whom Linux is completely alien. 

I can't write up a tutorial right now, as I'm at work, but if you just look up how to get into desktop mode on the steam deck if you're not familiar, and then google rimpy steam deck or rimpy Linux you'll find plenty of instructions.

**Comment by Tigeress4** [comment]
> > Steam deck is like Linux or how? Maybe isn't supported
> 
> Steam deck is Linux. It's most definitely supported, but you have to set it up through the desktop mode and it's not easy to do for someone to whom Linux is completely alien.
> 
> I can't write up a tutorial right now, as I'm at work, but if you just look up how to get into desktop mode on the steam deck if you're not familiar, and then google rimpy steam deck or rimpy Linux you'll find plenty of instructions.

Thank you, being dyslexic it makes it an extra layer of challenge for me. I can get into desktop mode easily enough now.  Still not sure enough to do all the extra steps. I forgot to send messages last week. Thank you both so much, slowly teaching myself new skills. :) Here's to continuing to learn at any age. 
Thank you again for pointing me in the right direction

**Comment by sparr** [comment]
@King-of-Blades On certain screen shapes/sizes, Releases are hidden at the bottom of the page, not up high in the sidebar.

**Comment by King-of-Blades** [comment]
> @King-of-Blades On certain screen shapes/sizes, Releases are hidden at the bottom of the page, not up high in the sidebar.

Well, if you're here to actually download rimpy, you're going to be currently using a PC, not a phone or some other device.

I would be willing to be a solid sum of money that the Venn diagram of people who are tech savvy enough to choose to use unorthodox aspect ratios and abnormal screen sizes on their home PC and people who are tech unsavvy enough to have no idea how to download something from GitHub is probably two separate circles.

**Comment by nukeop** [comment]
It's pretty crazy to download a 216MB program and just run it without seeing the source, especially for something as simple as a mod manager. Hosting this on Github without the source is a HUGE red flag. I don't blame people not trusting this, they have no reason to.

**Comment by nukeop** [comment]
Yeah, upon further inspection, it's a ton of random libraries, and a 102MB (!) binary, with no way to tell what it's going to do to my computer. Nope.
